.opspec/build/op.yml

```diff
@@ -14,7 +14,7 @@ run:
         ref: test
         inputs: { srcDir }
     - op:
-        ref: github.com/opspec-pkgs/_.op.bootstrap#2.2.0
+        ref: github.com/opspec-pkgs/_.op.bootstrap#2.2.1
```

Comment:
2.2.2?

Reply:
it's not published yet ; )

Semi confusing but this op uses itself in it's own build op.

Reply:
Ahh gotchaaa